### ISSUE TITLE
INCLUDE TYPE AS, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1916]
```abap
TYPES: BEGIN OF bar,
         field TYPE c LENGTH 1,
       END OF bar.

TYPES: BEGIN OF something,
         field1 TYPE c LENGTH 1.
         INCLUDE TYPE bar AS mo.
TYPES END OF something.

DATA moo TYPE something.
WRITE moo-field1.
WRITE moo-mo-field.
```

![image](https://user-images.githubusercontent.com/5888506/114839964-a020d300-9dd6-11eb-8905-a5be6f9a7166.png)
